Update vpc.md (#3311)
timescale/docs: use-timescale/vpc.md

inside the secure Timescale VPC. You control secure communication between apps in
your VPC and your Timescale Services using a dedicated Peering VPC. The AWS PrivateLink connecting 
Timescale VPC to the dedicated Peering VPC gives the same level of protection as using a direct AWS PrivateLink connection. 

> It only enables communication to be initiated from your Customer VPC to Timescale Services running

Reviewer: ```suggestion
This enables you to initiate unidirectional communication from your Customer VPC to Timescale Services running
```

Author: @billy-the-fish the suggestion leads to confusion. The communication will go in both directions (you make requests and get data back). But the communication can only be initiated (i.e. opening the connection) from the Customer VPC.

> in the Timescale VPC but not the other way around.

Reviewer: ```suggestion
in the Timescale VPC. Timescale cannot initiate communicate with your VPC. 
```
